app/help/en/faq.md

```diff
@@ -3,58 +3,1103 @@
 <a class="anchor" name="Table_of_Contents"></a>
 ## Table of Contents
 
+<a class="anchor" name="Table_of_Contents_General"></a>
+### General
 - <a href="/faq.html#What_is_Steemit_com">What is Steemit.com?</a>
+- <a href="/faq.html#How_does_Steemit_work">How does Steemit work?</a>
+- <a href="/faq.html#How_does_Steemit_differ_from_other_social_media_websites">How does Steemit differ from other social media websites?</a>
+- <a href="/faq.html#Does_it_cost_anything_to_post__comment__or_vote">Does it cost anything to post, comment, or vote?</a>
+- <a href="/faq.html#Can_I_earn_digital_tokens_on_Steemit_How">Can I earn digital tokens on Steemit? How?</a>
+- <a href="/faq.html#Where_do_the_tokens_come_from">Where do the tokens come from?</a>
+- <a href="/faq.html#Where_does_the_value_come_from">Where does the value come from?</a>
+- <a href="/faq.html#Why_are_people_getting_vastly_different_rewards">Why are people getting vastly different rewards?</a>
+
+<a class="anchor" name="Table_of_Contents_Accounts"></a>
+### Accounts
+- <a href="/faq.html#How_do_I_create_an_account">How do I create an account?</a>
+- <a href="/faq.html#What_information_do_I_need_to_provide_in_order_to_create_an_account">What information do I need to provide in order to create an account?</a>
+- <a href="/faq.html#Why_do_I_need_to_provide_my_email_and_phone_number">Why do I need to provide my email and phone number?</a>
+- <a href="/faq.html#Can_I_create_a_Steem_account_without_an_email_and_phone_number">Can I create a Steem account without an email and phone number?</a>
+- <a href="/faq.html#What_are_other_ways_to_create_an_account_on_the_blockchain_besides_using_Steemit_com">What are other ways to create an account on the blockchain besides using Steemit.com?</a>
+- <a href="/faq.html#It_is_not_letting_me_create_an_account_with_my_phone_number__What_should_I_do">It is not letting me create an account with my phone number. What should I do?</a>
+- <a href="/faq.html#What_happens_if_my_email_or_phone_number_changes">What happens if my email or phone number changes?</a>
+- <a href="/faq.html#Can_I_delete_or_deactivate_my_account">Can I delete or deactivate my account?</a>
+
+<a class="anchor" name="Table_of_Contents_Community"></a>
+### Community
+- <a href="/faq.html#Is_there_an_Etiquette_Guide_for_Steemit">Is there an Etiquette Guide for Steemit?</a>
+- <a href="/faq.html#Am_I_required_to_verify_my_identity">Am I required to verify my identity?</a>
+- <a href="/faq.html#What_are_whales__dolphins_and_minnows">What are whales, dolphins and minnows?</a>
```

Comment:
usage of the terms "whales", "dolphins", and "minnows" are not supported by steemit and we would like to avoid these terms in all official communications.